src/util/time.rs

```diff
@@ -16,21 +17,20 @@ pub fn time_now_us() -> u128 {
 
 /// Get the current UTC date and time as a string
 pub fn time_now_utc() -> String {
-    let time_now = OffsetDateTime::now_utc();
-    time_now.to_string()
+    let now_utc = Zoned::now().with_time_zone(TimeZone::UTC).timestamp();
+    now_utc.to_string()
```

Comment:
This can just be `Timestamp::now().to_string()`. You don't need to use `Zoned::now()` if you're just going to convert to UTC and get a timestamp.